stock_move_pivot_total_price/models/stock_move.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Bernat Obrador
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    product_total_price = fields.Float(
+        compute="_compute_product_total_price",
+        string="Total Product Price",
+        store=True,
+    )
+
+    @api.depends("product_id", "quantity")
+    def _compute_product_total_price(self):
+        for record in self:
+            if record.product_id and record.quantity:
```

Comment:
This if else is not really necessary. Both fields are required fields. It is OK, just extra processing on a busy table, but maybe it works out faster anyway if lots of 0's.